### PR TITLE
console.log support for multiple arguments

### PR DESCRIPTION
**Pull request by qwIvan**
such as `console.log('Hello', 'World', '!');`
image

### CODE REVIEW

File: js/editor-libs/console-utils.js [hunk 1]
```diff
@@ -86,6 +86,20 @@ module.exports = {
...
+    /**
+     * Concatenate the output of each argument with a blank space.
+     * @param args - Arguments of console.log
+     * @returns {*} Concatenated output as a string.
+     */
+    concatOutput: function(args) {
+        'use strict';
+        var output = '';
+        output = output + this.formatOutput(args[0]);
+        for (var i=1; i<args.length; i++) {
```
Comment (schalkneethling) [inline js/editor-libs/console-utils.js]: Code style nits: Change to

```
for (var i = 1, l = args.length; i < l; i++) {
```

File: js/editor-libs/console.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ module.exports = function() {
     };
 
     // eslint-disable-next-line no-console
-    console.log = function(loggedItem) {
-        consoleUtils.writeOutput(consoleUtils.formatOutput(loggedItem));
+    console.log = function() {
+        consoleUtils.writeOutput(consoleUtils.concatOutput(arguments));
```
Comment (schalkneethling) [inline js/editor-libs/console.js]: Perhaps the context is making this a little blurry but, it seems that this change will mean we loose the formatting done by `formatOutput`. That is definitely something we do not want to happen. Perhaps we should instead loop over the arguments, passing each one to the format function before concatenating together?

**Comment by schalkneethling** [comment]
@qwIvan Thank you for your contribution. Couple of items to review. I also want to note that, unless you change your current avatar, I will not in good faith be able to merge this pull request and add you as a contributor.

The image depicts something that I feel is disrespectful to woman, and this goes against out [code of conduct](https://github.com/mdn/interactive-examples/blob/master/CODE_OF_CONDUCT.md)

**Comment by qwIvan** [comment]
deleted concat function

**Comment by qwIvan** [comment]
deleted concat function

**Comment by qwIvan** [comment]
deleted concat function

**Comment by qwIvan** [comment]
deleted concat function

**Comment by schalkneethling** [comment]
@qwIvan Thank you for the updates. It is looking good. I have run into one problem where the value `undefined` is not output. Have not debugged it to see what is causing it. Feel free to look into that. I will also look into and let you know.

The example I saw that with is `weakmap-prototype-get.html`

**Comment by qwIvan** [comment]
Convert undefined to String and resolved.

**Comment by schalkneethling** [comment]
> Convert undefined to String and resolved.

That is indeed the simplest solution, I wonder though, do we want to output it as a string? It is not a `String` under the hood, and is generally not output as a string. With that said, the solution to this will add additional code(to loop over the final array and replace 'undefined' with `undefined` when writing the output) and complexity. I am wondering if it is worth the effort.

@wbamberg Your thoughts?

**Comment by wbamberg** [comment]
I see this:

```
// input
var f;
console.log("Hello", "World", f)
```

```
// output
> "Hello" "World" undefined
```

This *looks* fine, so unless there is some subtlety I am missing, I'm happy with it :).

**Pull request merged**